settings.php

```diff
@@ -8,6 +8,7 @@
 require "scripts/pi-hole/php/header.php";
 require "scripts/pi-hole/php/savesettings.php";
 require_once "scripts/pi-hole/php/FTL.php";
+require_once "scripts/pi-hole/php/func.php";
```

Comment:
You don't need to include this file again.
It is included by `header.php` and `savesettings.php`.